.github/workflows/build-insiders.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     needs: check_latest
     strategy:
       matrix:
-        platform: [windows-2019, macos-12, ubuntu-20.04]
+        platform: [windows-2019, macos-12, macos-14, ubuntu-20.04]
```

Comment:
Maybe have a comment here that says `macos-14` is for Apple Silicon builds since this isn't immediately obvious looking at the platform names.